lib/level10/games/game.ex

```diff
@@ -5,46 +5,144 @@ defmodule Level10.Games.Game do
   game's state, and then stored on the server to be updated or to serve data
   down to clients.
   """
+  # alias Level10.Games.{Card, GameRegistry, Player}
+  alias Level10.Games.Player
 
-  alias Level10.Games.{Card, Player}
-
+  @type join_code :: <<_::32>>
   @type t :: %__MODULE__{
-          complete: boolean(),
-          current_round: integer(),
-          discard_pile: list(Card.t()),
-          draw_pile: list(Card.t()),
-          hands: map(),
-          players: map(),
-          scoring: map(),
-          table: map()
+          current_round: non_neg_integer() | :pending | :completed,
```

Comment:
What would you think of splitting this into two parts? I like ditching the boolean, but I think we could do more of a state machine here that could be more useful for showing views. So we could have the `current_round` just be an integer, and then `current_stage` or something like that that could have `:lobby`, `:starting`, `:completed`, `:scorekeeping`, or `:playing` and we could then use that to determine which screen to show.

Reply:
I like that idea